package_manifest.yaml

```diff
@@ -42,46 +42,59 @@ copr_projects:
       - 'python39:3.9'
       - 'nodejs:12'
       - 'postgresql:12'
-    rhel_9_chroot: rhel-9-x86_64
-    rhel_8_chroot: rhel-8-x86_64
-    rhel_7_chroot: rhel-7-x86_64
+    rhel_9: '9'
+    rhel_8: '8'
+    rhel_7: '7'
```

Comment:
what's the benefit of having those numbers in variables?

Reply:
Mostly to prevent copy/paste errors in the future when we add RHEL 9 for example, or other client architectures. This shows up in so many places it could be easy to overlook.

Reply:
However, I am open to not doing it this way.

Reply:
I find it weird to read, but on the other hand, it doesn't hurt, so :shrug: